src/Wallet.purs

```diff
@@ -15,12 +17,21 @@ import Cardano.Types.Transaction
   , TransactionWitnessSet(TransactionWitnessSet)
   , Vkey(Vkey)
   , Vkeywitness(Vkeywitness)
+  , Utxo
+  , _vkeys
+  , TransactionOutput(..)
```

Comment:
> `(..)`

Reply:
Thanks, HLS keeps inserting .. rather than the name when I use an unimported constructor. I'll have to see if I can configure it